cmd/buildctl/build.go

```diff
@@ -315,7 +318,25 @@ func buildAction(clicontext *cli.Context) error {
 }
 
 func writeMetadataFile(filename string, exporterResponse map[string]string) error {
-	b, err := json.Marshal(exporterResponse)
+	out := make(map[string]interface{})
+	for key, value := range exporterResponse {
+		switch key {
+		case exptypes.ExporterImageDescriptorKey:
```

Comment:
I think lets just do this for all keys. If they decode properly then we show the original value.